plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -879,7 +879,7 @@ SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]\.php/cloud/users(?:/[^/]+)?$" \
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.value"
 
 # Fix Nextcloud Cookie FPs
-SecAction \
+SecRule REQUEST_FILENAME "@unconditionalMatch" \
```

Comment:
Can you add this comment above this rule?
```
# Operator @unconditionalMatch is used instead of a SecAction because of a bug
# in ModSecurity v3 which prevents SecActions to be removed using ctl action.
```